test/MemorySizeTest.cpp

```diff
@@ -288,6 +288,39 @@ TEST(MemorySize, Parse) {
                                                      {42_TB, "42 t"}},
       doTest);
 
+  // Check if string is truly optional
```

Comment:
I don't understand the comment. Do you mean check if the parsing is case-insensitive and then for the second block that the `B` is optional for `kB, TB, etc.`?